lib/actions-oclif/env/add.ts

```diff
@@ -171,31 +188,36 @@ async function setServiceVars(
 	options: FlagsDef,
 ) {
 	if (options.application) {
-		const serviceId = await getServiceIdForApp(
-			sdk,
-			options.application,
-			options.service!,
-		);
-		await sdk.models.service.var.set(serviceId, params.name, params.value!);
-	} else {
+		for (const app of options.application.split(',')) {
+			try {
+				const serviceId = await getServiceIdForApp(sdk, app, options.service!);
```

Comment:
If a comma-separated list of service names was allowed, then there could be an inner loop here over `options.service.split(',')`.